linux_os/guide/system/apparmor/all_apparmor_profiles_enforced/ansible/shared.yml

```diff
@@ -0,0 +1,20 @@
+# platform = multi_platform_sle
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+
+- name: Ensure all AppArmor Profiles are reloaded
+  ansible.builtin.shell: apparmor_parser -q -r /etc/apparmor.d/
+
+- name:  Ensure all AppArmor Profiles are enforcing
+  ansible.builtin.shell: aa-enforce /etc/apparmor.d/*
+
+- name: Collect unconfined processes 
+  ansible.builtin.shell: aa-unconfined
+  register: unconfined_processes
+
+- name: Provide details about unconfined processes 
+  ansible.builtin.shell: echo "The process {{ item }} may need to have a profile created or activated for them and then be restarted."
```

Comment:
I recommend to use the [assert](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/assert_module.html) module here.

Example:
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/firewalld_sshd_port_enabled/ansible/shared.yml#L80